lib/model.js

```diff
@@ -31,22 +32,43 @@ class SessionModel extends LocalModel {
     }
 
     getSessionData() {
-        let session = this.options.session;
+        let session = this.options.req.session;
         let key = this.options.key;
         let data = session[key] = session[key] || {};
         return data;
     }
 
     updateSessionData(changes) {
-        _.extend(this.options.session[this.options.key], changes);
+        let session = this.options.req.session;
+        _.extend(session[this.options.key], changes);
     }
 
     resetSessionData() {
-        this.options.session[this.options.key] = {};
+        let session = this.options.req.session;
+        session[this.options.key] = {};
+    }
+
+    save(cb) {
+        this.options.req.session.save(cb);
+    }
+
+    reload(cb) {
+        this.options.req.session.reload(err => {
+            if (err) return cb(err);
+            this._reload();
+            cb();
+        });
+    }
+
+    _reload() {
+        this.reset({ silent: true });
```

Comment:
Isn't `this.reset({ silent: true });` just setting localModel.attributes to a new object?
ie.
In https://github.com/UKHomeOffice/passports-model/blob/master/lib/local-model.js
doesn't it just set `this.attributes = {}`

If so, why not just do this?

Reply:
There is interplay between the reloading of the journey model and the session model - doing reload of the journey will also run _reload of the wizard model. (but we dont want to reload the session twice)